gpt.go

```diff
@@ -410,6 +410,34 @@ func StringToGuid(guid string) (res [16]byte, err error) {
 	return res, nil
 }
 
+// NewTable - return a valid empty Table for given sectorSize and diskSize
+//    Note that a Protective MBR is needed for lots of software to read the GPT table.
+func NewTable(sectorSize uint64, diskSize uint64, diskGuid Guid) Table {
```

Comment:
Hello scott.

Thanks for your patch :)

I suggest use diskSize as first positional arg and struct for all other args - for optional.
For example - if guid is empty - generate new random. If sector size is empty - use default 512 byte.

It allow for use set only need args + allow add new args for feature without lose compatible (for example add new GPT version if need, explicit partitions number).

Reply:
I didn't want to force the caller to pass a guid, but I also didn't want to add a dependency/import here.  I'll have to add a non-standard-library for uuid generation.

I don't personally feel that this is any more obvious or caller friendly than having the 2 explicit arguments.  It seems that it just "hides" them a little bit.  But I'm fine to let you make the call on that.

Reply:
I added a commit adjust to your feedback.
Thanks.